akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/JdbcOffsetStore.scala

```diff
@@ -60,17 +63,37 @@ private[akka] class JdbcOffsetStore[S <: JdbcSession](
       // init Statement in try-with-resource
       tryWithResource(conn.prepareStatement(settings.dialect.readOffsetQuery)) { stmt =>
 
+        stmt.setFetchSize(settings.fetchSize)
```

Comment:
do we need the fetchSize configuration or can we just rely on driver defaults? not expecting many rows or large values here

Reply:
yes, my point is that we can add it later if we see that there is a performance problem